cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -48,22 +56,37 @@ import qualified Cardano.Chain.Delegation as Delegation
 import qualified Cardano.Chain.Genesis as Genesis
 import           Cardano.Chain.Slotting (EpochNumber(..))
 import qualified Cardano.Chain.UTxO as UTxO
+
```

Comment:
I wouldn't say the intra-external-module import white space is necessary, but it's not critical.